### PR TITLE
Paperframe recipe needs less stuff / Paper bundle tweak

### PR DESCRIPTION
**Pull request by TsunamiAnt**
## About The Pull Request
~~Paper walls are very nice to look at and very fragile. AND THEY TAKE SO MUCH TIME TO MAKE IT'S UNBELIEVABLE.
You have to sit there and individually pull sheets of paper out of whatever you scrounged up or crafted.~~

~~A 5x5 room with 2 doors would take 50 paper frames. That means 200 sheets of paper. Now you have to sit there clicking and dropping paper 200 times, where do you get 200 paper from you ask?~~

~~At 30 paper a piece that's 7 paper bins or crafted natural paper stacks.~~

~~Good luck scrounging for those bins all around the station. Or waiting for the botanist to make wood(which you'll need anyways for the frames) If you wanna craft them instead you need 50 water and 1 wood per 30 paper by the way.~~

Ignore allat, we're changing it only a little bit now. 
Now the paper BUNDLE is called a bundle too, and has description text in bold noting that you can just cut the damn thing. Paper frames now need 2 wood, 10 paper instead cuz 20 really was a bit much.

## Why It's Good For The Game

Because the alternative is terrible, and it's super weak, and it looks good.

## Testing Photographs and Procedure

image

## Changelog
:cl:
tweak: paper frames now take 10 paper and 2 wood to craft
tweak: paper bundle is now also called that
add: add description text to paper bundle to show people you can just cut it open
/:cl:

### CODE REVIEW

**Comment by Tsar-Salat** [comment]
> tweak: paper frames take less stuff to craft

I mean, the number is right there, why omit it?

**Comment by TsunamiAnt** [comment]
fair

**Review by PestoVerde322 — APPROVED** [review]

**Comment by RDS88-beeman** [comment]
As much as I agree with this change, you can use a sharp object on crafted paper bundles to cut them open and spill their paper all over the floor. I don't know if there's something similar with paper trays, but for future, this should hopefully make pretty rooms easier.

**Comment by TsunamiAnt** [comment]
> As much as I agree with this change, you can use a sharp object on crafted paper bundles to cut them open and spill their paper all over the floor. I don't know if there's something similar with paper trays, but for future, this should hopefully make pretty rooms easier.

this actually changes everything, I'm gonna have to tweak this PR into a different direction...

File: code/modules/paperwork/paperbin.dm [hunk 1]
```diff
@@ -142,7 +142,7 @@
 
 /obj/item/paper_bin/bundlenatural
 	name = "natural paper bundle"
-	desc = "A bundle of paper created using traditional methods."
+	desc = "A bundle of paper created using traditional methods. <b>You can cut the cord on this with a sharp implement, freeing all 30 sheets at once.</b>"
```
Comment (EvilDragonfiend) [inline code/modules/paperwork/paperbin.dm]: This description should be added by `examine()` proc.

**Review by EvilDragonfiend — CHANGES_REQUESTED** [review]

File: code/modules/paperwork/paperbin.dm (hunk 1 above)
Comment (TsunamiAnt) [reply code/modules/paperwork/paperbin.dm]: Think I did it correctly

**Review by EvilDragonfiend — APPROVED** [review]

**Pull request merged**